pkg/apis/apps/validation/validation_test.go

```diff
@@ -24,6 +24,7 @@ import (
 	"github.com/davecgh/go-spew/spew"
 	"github.com/google/go-cmp/cmp"
 	"github.com/google/go-cmp/cmp/cmpopts"
+	"github.com/kubernetes/utils/blob/master/pointer/pointer.go"
```

Comment:
```suggestion
	"k8s.io/utils/pointer"
```

I think that is correct..

Reply:
Thank you for the point in the right direction!